redhat-cop/tower_utilities
# Tower install ocp

### What does this PR do?
Adds a role to deploy a containerised Ansible Tower instance with optional external database.

### How should this be tested?
Please follow the README in regards to running the new role. If successful the playbook will run through and you should see Ansible tower in Openshift.

### Is there a relevant Issue open for this?
closes #6 

### Other Relevant info, PRs, etc.
N/A
